src/script/acquisition-sdk.ts

```diff
@@ -58,14 +58,15 @@ export class AcquisitionStatus {
 }
 
 export class AcquisitionManager {
+    private readonly BASER_URL = "https://codepush.appcenter.ms"
```

Comment:
no need, we already have it defined. Consider a filter by "appcenter.ms", because there are two possible endpoint urls for App Center CodePush service.